packages/core/upload/admin/src/pages/App/MediaLibrary.js

```diff
@@ -127,6 +127,15 @@ export const MediaLibrary = () => {
     setShowEditFolderDialog((prev) => !prev);
   };
 
+  useEffect(() => {
+    if (assetCount > 0) {
+      setQuery({
+        ...query,
+        assetCount,
```

Comment:
Mh, I'm not a fan of this way to pass around state tbh. I can think of two ways to solve that problem:

1. Call a callback after `handleConfirmRemove()` was called and then do the redirection. In `MediaLibrary` you do have the total asset count available.
2. Pass down the total as a prop to `BulkDeleteButton`

I think I'd prefer the first one. It doesn't hide the redirect in the delete button ... Wdyt?

Reply:
Yeah I think I would try to implement it in the first way but I think we'd also need to change the tests too